samza-core/src/main/scala/org/apache/samza/container/TaskInstance.scala

```diff
@@ -531,8 +563,11 @@ class TaskInstance(
                 "Saved exception under Caused By.", commitException.get())
             }
           } else {
+            commitExceptionCounter.set(0)
+            commitTimeoutCounter.set(0)
             metrics.commitAsyncNs.update(System.nanoTime() - asyncStageStartNs)
             metrics.commitNs.update(System.nanoTime() - commitStartNs)
+            metrics.lastAsyncCommitNs.set(System.nanoTime())
```

Comment:
Do we need this metric ? We already have `commitAsyncNs` so hard to reason why we need this one.

Reply:
They are separate metrics.
- `commitAsyncNs` is the duration between `asyncStageStartNs` and current time.
- `lastAsyncCommitNs` is the timestamp of the most recent commit

Reply:
Since we are adding configs here to skip commit exception, this metric might have some utility in viewing when was the last commit. I would rather name it as `lastCommitTimestampNs`.
Metrics like `commitNs`, `processns`, `windowns` have been used to quantify the time taken for each of these steps. `lastAsyncCommitNs` is too similar to `commitAsyncNs` so let's rename the metric.

Reply:
Renamed.